modules/ComscoreStreamingTag/resources/mw.ComscoreStreamingTag.js

```diff
@@ -0,0 +1,579 @@
+		playerVersion: mw.getConfig('version'),
+		genericPluginUrlSecure: "https://sb.scorecardresearch.com/c2/plugins/streamingtag_plugin_generic.js",
+		genericPluginUrl: "http://b.scorecardresearch.com/c2/plugins/streamingtag_plugin_generic.js",
+
+		bindPostfix: '.ComScoreStreamingTag',
+		moduleName: 'ComScoreStreamingTag',
+		unknownValue: 'unknown',
+
+		genericPlugin: null, // Placeholder reference for comScore Generic plugin
+
+		clipNumberMap: {},
+	    clipNumberCounter: 0,
+		playerEvents: null,
+		inFullScreen: false,
+		currentPlayerPluginState: null,
+		bandwidth: 0,
+		lastCuePointUpdateTime: -1,
+		adCuePoints: null,
+		adsPlayed: [],
+		currentAd: {},
+		currentBitrate: 0,
+		shouldSetClip: true,
+
+		PlayerPluginState: function () {
+			var stringMap = [ "initializing", "idle", "new_clip_loaded", "playing", "paused", "ended_playing", "buffering", "seeking", "scrubbing", "ad_playing", "ad_paused", "ad_ended_playing", "destroyed"];
+			return {
+				INITIALIZING: 0,
+				IDLE: 1,
+				NEW_CLIP_LOADED: 2,
+				PLAYING: 3,
+				PAUSED: 4,
+				ENDED_PLAYING: 5,
+				BUFFERING: 6,
+				SEEKING: 7,
+				SCRUBBING: 8,
+				AD_PLAYING: 9,
+				AD_PAUSED: 10,
+				AD_ENDED_PLAYING: 11,
+				DESTROYED: 13,
+				toString: function (eventType) {
+					return stringMap[eventType];
+				}
+			};
+		},
+
+		init: function( embedPlayer, callback ){
+			this.embedPlayer = embedPlayer;
+			this.setupConfig();
+			this.currentPlayerPluginState = this.PlayerPluginState().INITIALIZING;
+			var _this = this;
+			var _callback = callback;
+
+			this.mediaElement = new mw.MediaElement(embedPlayer);
+
+			_this.currentAd.id = "";
+			_this.currentAd.type = "";
+			_this.currentAd.index = 0;
+			_this.currentAd.duration = 0;
+
+			var comScoreSettings = {};
+			if (_this.isSecure())
+				comScoreSettings.secure = true;
+
+			// Settings displayed in the interface
+			if (_this.config.c2)
+				comScoreSettings.c2 = _this.config.c2;
+			//if (_this.config.labelMapping)
+			//comScoreSettings.labelmapping = [];
+			if (_this.config.pageView)
+				comScoreSettings.pageview = _this.config.pageView;
+			if (_this.config.logUrl)
+				comScoreSettings.logurl = _this.config.logUrl;
+			if (_this.config.persistentLabels)
+				comScoreSettings.persistentlabels = _this.config.persistentLabels;
+
+			// Not displayed in the interface
+			if (_this.config.debug)
+				comScoreSettings.debug = _this.config.debug;
+			if (_this.config.include)
+				comScoreSettings.include = _this.config.include;
+			if (_this.config.includePrefixes)
+				comScoreSettings.include_prefixes = _this.config.includePrefixes;
+			if (_this.config.exclude)
+				comScoreSettings.exclude = _this.config.exclude;
+			if (_this.config.excludePrefixes)
+				comScoreSettings.exclude_prefixes = _this.config.excludePrefixes;
+
+			var standalonePluginUrl = this.isSecure() ? this.genericPluginUrlSecure : this.genericPluginUrl;
+			$.getScript(standalonePluginUrl, function(){
```

Comment:
I would recommend kWidget.appendScriptUrl .. https://github.com/kaltura/mwEmbed/blob/master/kWidget/kWidget.js#L2061 

this is because it will not do the script cache breaker stuff that jQuery does by default. You otherwise need to update the AJAX call to not disable caching: 
http://api.jquery.com/jQuery.getScript/#caching-requests